fredlexer.py

```diff
@@ -0,0 +1,140 @@
+"""
+    FRED lexer derived from the pygments.lexers.RegexLex class
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for the FRED Modeling Language (TM).
+
+    :copyright: Copyright 2021 by Epistemix In.c
+    :license: BSD, see LICENSE for details.
+"""
```

Comment:
@jchrszcz did you write a program for FRED syntax highlighting around November time? A Sublime text plugin perhaps? If so perhaps we could figure out a way to have it share lexer configuration with this script so there's a single source of truth. This will help when the language changes

Reply:
I did! It is [here](https://github.com/Epistemix-com/data_science_team/blob/master/tools/fred.sublime-syntax).

Some quick googling suggests Sublime doesn't play nicely with lexer, but I'm sure we could build out FREDLexer and automate parsing it into the format required for Sublime.